src/forms/TextInput.jsx

```diff
@@ -38,13 +45,26 @@ const TextInput = (props) => {
 		labelClassName
 	);
 
+	const iconSize = props.iconSize || 'xs';
```

Comment:
you can pull `iconSize` out above with the other props (line ~29)

Reply:
Keeping as-is in the interest of not having to define a new variable for icon size